### ISSUE TITLE
Improve Presentation of Feed Information

### ISSUE THREAD

**Issue by edasmalchi** [cal-itp/reports#139]
As a stakeholder reading the report, it’s unclear what this info refers to and where it fits in the hierarchy of the page. Additionally, the lack of proper HTML headers is an accessibility issue.

**Action ideas:**
- [ ] Orient this section horizontally, above charts and separated by existing gold section breaks
- [ ] Strike "Feed Published" and add H2 “Current GTFS Feed Information” for this section
- [ ] Below that, include each existing element but with human-readable dates and updated labels
    - [ ] Feed End Date should be human-readable
    - [ ] "Days with no service" should probably be something like "Days in {month} With No Service" (it's a bit tricky since feed end date refers to the feed at a whole, but this is based on a query for the current month)

<img width="1399" alt="Screen Shot 2022-05-09 at 9 03 39 AM" src="https://user-images.githubusercontent.com/54728455/167451846-579ec3b4-e889-416f-ae53-86f58ebfb34b.png">
